packages/app/src/app/pages/Sandbox/Editor/Workspace/Project/SandboxConfig/TemplateConfig.tsx

```diff
@@ -70,6 +84,25 @@ export const TemplateConfig = observer(() => {
           )}
         </PropertyValue>
       </Item>
+      <Item>
+        <PropertyName>Public</PropertyName>
+        <PropertyValue
+          css={`
```

Comment:
Hmm, can we move everything with an inlined style to `elements.ts`?

Reply:
Fixed